app/src/main/java/com/jerboa/ui/components/community/list/CommunityListActivity.kt

```diff
@@ -49,6 +57,8 @@ fun CommunityListActivity(
     siteViewModel: SiteViewModel,
     blurNSFW: Boolean,
     drawerState: DrawerState,
+    appSettingsViewModel: AppSettingsViewModel,
```

Comment:
Seems pointless to bring in the entire appSettingsViewModel here. Just add `saveSearchHistory` as a param.